apps/tutorial-servo-can-control/TutorialServo.cpp

```diff
@@ -0,0 +1,45 @@
+
+#include "mbed.h"
+#include "TutorialServo.h"
+
+#include <iostream>
+using namespace std;
+
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees = 180.0,
+float minPulsewidthInMs = 1, float maxPulsewidthInMs = 2)
+{
+    //declares this pin as pwm using pwmout.
```

Comment:
Inside your constructor there doesn't need to be any code. Really in this application the only purpose of the constructor is to use the member initializer list to initialize the member variables of the object.